### ISSUE TITLE
synchronization via rest backend not working on oracle.

### ISSUE THREAD

**Issue by gochev** [nemesis-software/nemesis-support#16]
Currently in 1.5.1 the synchronization of single entity doesnt work.

```
GET ..../rest/backend/synchronize?entityName=cms_slot&id=43349714143056960 leads to 
{
  "timestamp" : 1499257614777,
  "status" : 500,
  "error" : "Internal Server Error",
  "exception" : "org.springframework.transaction.CannotCreateTransactionException",
  "message" : "Could not open JPA EntityManager for transaction; nested exception is java.sql.SQLException: READ_COMMITTED and SERIALIZABLE are the only valid transaction levels",
  "path" : "/rest/backend/synchronize"
}
```
![screen shot 2017-07-05 at 15 26 40](https://user-images.githubusercontent.com/5537142/27864280-970db786-6196-11e7-9676-34a3b570c2d9.jpg)

**Comment by ptahchiev** [comment]
Can you please provide the stacktrace from the server?

**Comment by gochev** [comment]
This one is also fixed as part of 1.5.2.1
